bolero/environment/optimum_trajectory.py

```diff
@@ -294,5 +294,116 @@ def plot(self, ax):
+        List of obstacles.
+
+    obstacle_dist : float, optional (default: 0.1)
+        Distance that should be kept to the obstacles (penalty is zero outside
+        of this area)
+
+    curbing_obstacles : float, optional (default: 0)
+        Slow down the move if closer than *obstacle_dist* to an obstacle.
+        Multiple obstacles in the vicinity increase the effect.
+
+    penalty_start_dist : float, optional (default: 0)
+        Penalty weight for distance to start at the beginning
+
+    penalty_goal_dist : float, optional (default: 0)
+        Penalty weight for distance to goal at the end
+
+    penalty_vel : float, optional (default: 0)
+        Penalty weight for velocities
+
+    penalty_acc : float, optional (default: 0)
+        Penalty weight for accelerations
+
+    penalty_obstacle : float, optional (default: 0)
+        Penalty weight for obstacle avoidance
+
+    log_to_file: optional, boolean or string (default: False)
+        Log results to given file, it will be located in the $BL_LOG_PATH
+
+    log_to_stdout: optional, boolean (default: False)
+        Log to standard output
+    """
+    def __init__(self, x0=np.zeros(2), g=np.ones(2), execution_time=1.0,
+                 dt=0.01, obstacles=None, obstacle_dist=0.1, curbing_obstacles=0,
+                 penalty_start_dist=0.0, penalty_goal_dist=0.0,
+                 penalty_vel=0.0, penalty_acc=0.0, penalty_obstacle=0.0,
+                 log_to_file=False, log_to_stdout=False):
+        super(OptimumTrajectoryCurbingObstacles, self).__init__(
+            x0=x0, g=g, execution_time=execution_time, dt=dt, obstacles=obstacles,
+            obstacle_dist=obstacle_dist, penalty_start_dist=penalty_start_dist,
+            penalty_goal_dist=penalty_goal_dist, penalty_vel=penalty_vel, penalty_acc=penalty_acc,
+            penalty_obstacle=penalty_obstacle, log_to_file=log_to_file, log_to_stdout=log_to_stdout)
+        self.curbing_obstacles = curbing_obstacles
+        self.damping = 0
+
+    def init(self):
+        """Initialize environment."""
+        super(OptimumTrajectoryCurbingObstacles, self).init()
+        self.X *= np.nan
+        self.Xd *= np.nan
+        self.Xdd *= np.nan
+
+    def reset(self):
+        """Reset state of the environment."""
+        super(OptimumTrajectoryCurbingObstacles, self).reset()
+        self.damping = 0
+        self.X *= np.nan
+        self.Xd *= np.nan
+        self.Xdd *= np.nan
+
+    def set_inputs(self, values):
+        """Set environment inputs, e.g. next action.
+
+        Parameters
+        ----------
+        values : array,
+            Inputs for the environment: positions, velocities and accelerations
+            in that order, e.g. for n_task_dims=2 the order would be xxvvaa
+        """
+        if self.damping:
+            # can only occur after step action, thus self.t > 0
+            if self.t == 0:
+                raise ValueError
```

Comment:
You would raise a RuntimError here. Also please use syntax that is compatible to Python 3: `raise RuntimeError("bla bla bla")`

Reply:
Is it just "bla bla" is python 2? ;-)
OK, I'll fix it.